pages/resources/channel.mdx

```diff
@@ -485,6 +487,19 @@ An object that specifies the emoji to use as the default way to react to a `GUIL
 
 ^1^ At most one of `emoji_id` and `emoji_name` may be set to a non-null value.
 
+### Icon Emoji Object
+
+An object that specifies the emoji to use as the icon displayed next to a channel's name.
+
+###### Icon Emoji Structure
+
+| Field          | Type       | Description                        |
+| -------------- | ---------- | ---------------------------------- |
+| emoji_id ^1^   | ?snowflake | The ID of a guild's custom emoji   |
+| emoji_name ^1^ | ?string    | The unicode character of the emoji |
+
+^1^ At most one of `emoji_id` and `emoji_name` may be set to a non-null value.
```

Comment:
No need to make this a footnote if it applies to all of the fields.

Reply:
I copy/pasted it from the `Default Reaction Object` section which is already written that way. If we change this here do we change it in both places?

Reply:
Is this even true here? I remember this field having some weird behavior (e.g. I'm pretty sure the entire object can be null AND both fields in the object can be null and both cases result in no icon)

Reply:
The [official documentation](https://discord.com/developers/docs/resources/channel#default-reaction-object) says "Exactly one of `emoji_id` and `emoji_name` must be set", and that also seems to be the error returned by the api:

![image](https://github.com/discord-userdoccers/discord-userdoccers/assets/9750071/37cf4757-6b47-438e-9139-2334a111322d)

Reply:
Setting both to `null`, as well as the entire field to `null` works too

Reply:
Now that I think about it, can this object be merged with the above one into something more generic that can be used for both cases?

Reply:
Probably, but then we have to come up with a name for it which fits both uses and isn't confusing at the same time (so ex. can't be "Emoji Object"), and I don't think I have any ideas xd

Reply:
partial emoji object is probably fine